app/testOnly/formsmodel/StartJourneyForm.scala

```diff
@@ -64,7 +66,11 @@ object StartJourneyForm {
         "signInAs" -> signInMapping,
         "enrolments" -> enrolmentsMapping,
         "origin" -> originMapping,
-        "eligibilityErrors" -> seq(enumeratum.Forms.enum(EligibilityErrors))
+        "eligibilityErrors" -> seq(enumeratum.Forms.enum(EligibilityErrors)),
+        "debtTotalAmount" -> Forms.of(amountOfMoneyFormatter(
+          isTooSmall = AmountInPence(100) > AmountInPence(_),
+          isTooLarge = AmountInPence(_) > AmountInPence(1000000)
```

Comment:
is the amount for too large not going to be dependent on regime? I see this equates to £10k yet PAYE does allow debts up to £15k - just a comment given the test only nature of the page

Reply:
True, sorry I just put in a random value based on SA (which was 10k last time I worked on it), will update this to 15k. Also when more regimes come along we can remove this/parameterise it